proto/api_v3/query_service.proto

```diff
@@ -43,6 +43,7 @@ message SpansResponseChunk {
 // Query parameters to find traces.
 // Note that some storage implementations do not guarantee the correct implementation of all parameters.
 message TraceQueryParameters {
+  // Service name as shown in the frontend. Required.
```

Comment:
I don't think "in the frontend" is a useful definition. Service Name is defined in the data model: `Span.Process.serviceName`

Reply:
Fine for me. I'd just prefer to have some details here :)